bluemira/equilibria/equilibrium.py

```diff
@@ -1048,17 +1068,19 @@ def set_vcontrol(self, vcontrol: Optional[str] = None):
         vcontrol:
             Vertical control strategy
         """
-        if vcontrol == "virtual":
-            self.controller = VirtualController(self, gz=2.2)
-        elif vcontrol == "feedback":
-            raise NotImplementedError
-        elif vcontrol is None:
+        if vcontrol is None:
```

Comment:
I would change this varible name to vcontrol_str

And change the others to vcontrol_type